autonomy/data/Dockerfiles/dev/Pipfile

```diff
@@ -16,6 +16,8 @@ open-aea-cli-ipfs = {editable = true, path = "/open-aea/plugins/aea-cli-ipfs"}
 open-aea-ledger-cosmos = {editable = true, path = "/open-aea/plugins/aea-ledger-cosmos"}
 open-aea-ledger-ethereum = {editable = true, path = "/open-aea/plugins/aea-ledger-ethereum"}
 py_eth_sig_utils = {version = "==0.4.0"}
+pytest = {version = ">=7.0.0,<7.2.0"}
+coverage = {version = ">=6.4.4,<7.0.0"}
```

Comment:
Since we use editable installation for frameworks we don't install every dependency listed in setup.py so we'll have to install these dependencies from the Pipfile

Reply:
This is the reason the dev mode is not working atm

Reply:
Are you sure this is about it being editable and not the fact that `extras=[all]` is not specified?

Reply:
I'll check

Reply:
Fixed